MigrationExecutorApp/ChangeFeedProcessortHost.cs

```diff
@@ -108,92 +121,131 @@ public async Task CloseAsync()
-                CollectionName = this.config.MonitoredCollectionName
-            };
 
-            var policy = new ConnectionPolicy()
+            var docTransformer = new DefaultDocumentTransformer();
+          
+            if (!String.IsNullOrEmpty(config.BlobConnectionString))
             {
-                ConnectionMode = ConnectionMode.Direct,
-                ConnectionProtocol = Protocol.Tcp
-            };
-
-            policy.PreferredLocations.Add("North Europe");
+                BlobServiceClient blobServiceClient = new BlobServiceClient(config.BlobConnectionString);
+                containerClient = blobServiceClient.GetBlobContainerClient(config.BlobContainerName);
+                await containerClient.CreateIfNotExistsAsync();
+            }
 
-            // lease collection info 
-            var leaseCollectionLocation = new DocumentCollectionInfo
+            DateTime starttime = DateTime.MinValue.ToUniversalTime();
+            if (config.DataAgeInHours.HasValue)
             {
-                Uri = new Uri(this.config.LeaseUri),
-                MasterKey = this.config.LeaseSecretKey,
-                DatabaseName = this.config.LeaseDbName,
-                CollectionName = this.config.LeaseCollectionName,
-                ConnectionPolicy = policy
-            };
-
-            // destination collection info 
-            var destCollInfo = new DocumentCollectionInfo
-            {
-                Uri = new Uri(this.config.DestUri),
-                MasterKey = this.config.DestSecretKey,
-                DatabaseName = this.config.DestDbName,
-                CollectionName = this.config.DestCollectionName
-            };
-
-            var processorOptions = new ChangeFeedProcessorOptions();
-            if(config.DataAgeInHours.HasValue) {
                 if (config.DataAgeInHours.Value >= 0)
                 {
-                    processorOptions.StartTime = DateTime.UtcNow.Subtract(TimeSpan.FromHours(config.DataAgeInHours.Value));
+                    starttime = DateTime.UtcNow.Subtract(TimeSpan.FromHours(config.DataAgeInHours.Value));
                 }
-            } else
-            {
-                processorOptions.StartFromBeginning = true;
             }
 
-            processorOptions.LeaseRenewInterval = TimeSpan.FromSeconds(30);
+            changeFeedProcessor = sourceCollectionClient.GetContainer(config.MonitoredDbName, config.MonitoredCollectionName)
+                .GetChangeFeedProcessorBuilder<Document>("Live Data Migrator", ProcessChangesAsync)
+                .WithInstanceName("consoleHost")
+                .WithLeaseContainer(sourceCollectionClient.GetContainer(config.LeaseDbName, config.LeaseCollectionName))
+                .WithStartTime(starttime)               
+                .WithMaxItems(1000)
+                .Build();
 
-            Trace.TraceInformation("Processor options Starts from Beginning - {0}, Lease renew interval - {1}",
```

Comment:
Missing Trace on new code?

Reply:
this was for tracing items in ChangeFeedProcessorOptions, for which I can't see equivalent in V3